admin-configmanager/app/.npmrc

```diff
@@ -0,0 +1,2 @@
+@spectrum:registry=https://artifactory.corp.adobe.com:443/artifactory/api/npm/npm-spectrum-release/
```

Comment:
I don't think this should be here. We need a more official way to deal with the private nature of React Spectrum. I think the Spectrum CSS team has a working model for including the packages we need while also marking them as not open source.

Reply:
ok I'll take out for now